backend/app/controllers/spree/admin/products_controller.rb

```diff
@@ -25,6 +25,11 @@ def update
         if params[:product][:option_type_ids].present?
           params[:product][:option_type_ids] = params[:product][:option_type_ids].split(',')
         end
+        if updating_variant_property_rules?
+          params[:product][:variant_property_rules_attributes].each do |index, param_attrs|
+            param_attrs[:option_value_ids] = param_attrs[:option_value_ids].split
```

Comment:
Maybe it should be a `.join(',')`? I'm not used to seeing space delimited params in rails